cypress/integration/fileMenu.cy.js

```diff
@@ -0,0 +1,153 @@
+import { typeInput } from '../helpers/common.js'
+import { selectEventProgram } from '../helpers/dimensions.js'
+import { clickMenubarUpdateButton } from '../helpers/menubar.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const ITEM_NEW = 'file-menu-new'
+const ITEM_OPEN = 'file-menu-open'
+const ITEM_SAVE = 'file-menu-save'
+const ITEM_SAVEAS = 'file-menu-saveas'
+const ITEM_RENAME = 'file-menu-rename'
+const ITEM_TRANSLATE = 'file-menu-translate'
+const ITEM_SHARING = 'file-menu-sharing'
+const ITEM_GETLINK = 'file-menu-getlink'
+const ITEM_DELETE = 'file-menu-delete'
+
+const defaultItemsMap = {
+    [ITEM_NEW]: false,
```

Comment:
Shouldn't `NEW` and `OPEN` always be `true`? They are indeed the "default" state as that's what you get without making any selection at all..

Reply:
Updated